translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"realizar \"get\", \"watch\" e \"list\" em pods\n"
+"\t\tkubectl create clusterrole pod-reader —verb=get,list,watch —"
+"resource=pods\n"
+"\n"
+"\t\t# Cria a ClusterRole com o nome de \"pod-reader\" com um ResourceName "
+"especificado\n"
+"\t\tkubectl create clusterrole pod-reader —verb=get,list,watch —"
+"resource=pods —resource-name=readablepod"
+
+#: pkg/kubectl/cmd/create_role.go:41
+msgid ""
+"\n"
+"\t\t# Create a Role named \"pod-reader\" that allows user to perform \"get"
+"\", \"watch\" and \"list\" on pods\n"
+"\t\tkubectl create role pod-reader --verb=get --verb=list --verb=watch --"
+"resource=pods\n"
+"\n"
+"\t\t# Create a Role named \"pod-reader\" with ResourceName specified\n"
+"\t\tkubectl create role pod-reader --verb=get --verg=list --verb=watch --"
+"resource=pods --resource-name=readablepod"
+msgstr ""
+"\n"
+"\t\t# Cria uma Role com o nome de \"pod-reader\" que permite o usuário "
+"realizar \"get\", \"watch\" e \"list\" em pods\n"
+"\t\tkubectl create role pod-reader —verb=get —verb=list —verb=watch —"
+"resource=pods\n"
+"\n"
+"\t\t# Cria uma Role com o nome de \"pod-reader\" com um ResourceName "
+"especificado\n"
+"\t\tkubectl create role pod-reader —verb=get —verg=list —verb=watch —"
+"resource=pods —resource-name=readablepod"
+
+#: pkg/kubectl/cmd/create_quota.go:35
+msgid ""
+"\n"
+"\t\t# Create a new resourcequota named my-quota\n"
+"\t\tkubectl create quota my-quota --hard=cpu=1,memory=1G,pods=2,services=3,"
+"replicationcontrollers=2,resourcequotas=1,secrets=5,"
+"persistentvolumeclaims=10\n"
+"\n"
+"\t\t# Create a new resourcequota named best-effort\n"
+"\t\tkubectl create quota best-effort --hard=pods=100 --scopes=BestEffort"
+msgstr ""
+"\n"
+"\t\t# Cria um novo resourcequota com o nome de my-quota\n"
+"\t\tkubectl create quota my-quota —hard=cpu=1,memory=1G,pods=2,services=3,"
+"replicationcontrollers=2,resourcequotas=1,secrets=5,"
+"persistentvolumeclaims=10\n"
+"\n"
+"\t\t# Cria um novo resourcequota com o nome de best-effort\n"
+"\t\tkubectl create quota best-effort —hard=pods=100 —scopes=BestEffort"
+
+#: pkg/kubectl/cmd/create_pdb.go:35
+#, c-format
+msgid ""
+"\n"
+"\t\t# Create a pod disruption budget named my-pdb that will select all pods "
+"with the app=rails label\n"
+"\t\t# and require at least one of them being available at any point in "
+"time.\n"
+"\t\tkubectl create poddisruptionbudget my-pdb --selector=app=rails --min-"
+"available=1\n"
+"\n"
+"\t\t# Create a pod disruption budget named my-pdb that will select all pods "
+"with the app=nginx label\n"
+"\t\t# and require at least half of the pods selected to be available at any "
+"point in time.\n"
+"\t\tkubectl create pdb my-pdb --selector=app=nginx --min-available=50%"
+msgstr ""
+"\n"
+"\t\t# Cria um pod disruption budget com o nome de my-pdb que irá selecionar "
+"todos os pods com o label app=rails\n"
+"\t\t# e requer pelo menos um deles disponível em qualquer momento.\n"
```

Comment:
I think here it is more clear "e requer que pelo menos um deles esteja disponível a qualquer momento"